packages/manager/core/generator/app/templates/src/tracking.constant.ts.hbs

```diff
@@ -0,0 +1,4 @@
+export const LEVEL2 = '86';
```

Comment:
**nitpick:** `LEVEL2` depends on the universe of the app and region

For example,
1. PCI apps has "86" for EU/CA and "56" for US.
2. Baremetal Cloud apps has "85" for EU/CA and "57" for US.
3. Hub has "88" for EU/CA/US
4. Web apps has "84" for EU/CA.
5. Telecom apps has "87" for EU.

So this cannot be a constant. Refer https://github.com/ovh/manager/blob/596170e7b496f367ad75ab8273e6308d27e7b7a3/packages/manager/apps/dedicated/client/app/at-internet.constants.js for more info.

Reply:
Sure I understand your POV, but why in the code we have this following condition ?

https://github.com/ovh/manager/blob/630c145c9b1a6678f80a4dd14af4d73e5ac384fd/packages/components/ovh-at-internet/src/ovh-at-internet.ts#L62

I created a constant cause I see in anoher project that the level2 attribute was harcoded directly in trackPage or trackClick, so I putted it in the constant variable file, like core team in PCI

Here an exemple https://github.com/ovh/manager/blob/630c145c9b1a6678f80a4dd14af4d73e5ac384fd/packages/manager/modules/telecom-dashboard/src/index.js#L93

What did you suggest to us to tackle this topic ?

Reply:
You might see the hard-coded values because Core stack team has added apps only on PCI so they hard-code `86` and the other example you have given is for telecom app and it has `87`.

But we cannot hard-code the value from the generator instead we should either accept it as an input while generating the app or set the value from the user's universe input.

Reply:
Ok, I add a command for asking the 'level2' 👍